CONTRIBUTING.md

```diff
@@ -29,6 +29,16 @@ If you wish to contribute code, you must sign the [contributor license agreement
 # Open Development
 All contributors follow GitHub’s pull request (PR) workflow to submit changes. Before merging, all pull requests require two approvals.
 
+# Structure of the repository
+
+Our repository is organized as a monorepo, meaning it houses multiple [projects](https://github.com/vmware/versatile-data-kit/tree/main/projects) and [plugins](https://github.com/vmware/versatile-data-kit/tree/main/projects/vdk-plugins) within the same repository.
+
+You can find each [individual project](https://github.com/vmware/versatile-data-kit/tree/main/projects) and [plugin](https://github.com/vmware/versatile-data-kit/tree/main/projects/vdk-plugins) located in their respective directories.
+The design of our monorepo allows for each project and plugin to be developed independently.
```

Comment:
There's probably an edge case where this isn't true. Still, it's not a huge pain to open two projects side by side, so I mostly agree.